config/cesm/machines/config_machines.xml

```diff
@@ -1094,6 +1094,118 @@
     </environment_variables>
   </machine>
 
+  <machine MACH="lawrencium-lr3">
+    <DESC>Lawrencium LR3 cluster at LBL, OS is Linux (intel), batch system is SLURM</DESC>
+    <NODENAME_REGEX>n00*</NODENAME_REGEX>
+    <OS>LINUX</OS>
+    <COMPILERS>intel</COMPILERS>
+    <MPILIBS>openmpi,mpi-serial</MPILIBS>
+    <CIME_OUTPUT_ROOT>/global/scratch/$ENV{USER}</CIME_OUTPUT_ROOT>
+    <DIN_LOC_ROOT>/global/scratch/$ENV{USER}/cesm_input_datasets/</DIN_LOC_ROOT>
```

Comment:
Ideally you want DIN_LOC_ROOT in a place not associated with $USER so that everyone who uses cesm on the system has access to it.   This is also true of BASELINE_ROOT and CCSM_CPRNC.

Reply:
I can look into this more, but I'm not aware of any projects with large space allocations on lawrencium that could hold these files, but maybe there are.  As far as I know, some of the data atmosphere datasets are getting pretty large.